src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -311,6 +313,14 @@ private void regenerateAITasks() {
 				L10N.t("elementgui.living_entity.condition_natural_spawn"), VariableTypeLoader.BuiltInTypes.LOGIC,
 				Dependency.fromString("x:number/y:number/z:number/world:world")).setDefaultName(
 				L10N.t("condition.common.use_vanilla")).makeInline();
+		bodyVisibleCondition = new ProcedureSelector(this.withEntry("entity/condition_is_body_visible"), mcreator,

```

Comment:
I highly suspect these conditions are client-side only, so this should be indicated here

Reply:
They are effectively client-side as they are only visual effects.

Reply:
Then these condition selectors should show they are client side to not confuse users why eg. NBT entity data is not syncing here